u8timeseries/tests/test_models.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+import pandas as pd
+import numpy as np
+
+from ..timeseries import TimeSeries
+from u8timeseries import Prophet, KthValueAgoBaseline, ExponentialSmoothing, TimeSeries, Arima, AutoArima
+from u8timeseries import StandardRegressiveModel
+from u8timeseries.models.theta import Theta
+
+class ModelsTestCase(unittest.TestCase):
+    __test__ = True
+
+    # general prediction / training parameters
+    forecasting_horizon = 5
+    regression_window = 5
+
+    # dummy timeseries for autoregression forecasting and target for regression
+    times = pd.date_range('20000101', '20000130')
+    values = np.sin(range(len(times))) + np.array([2.0] * len(times))
+    ts: TimeSeries = TimeSeries.from_times_and_values(times, values)
+
+    # dummy feature timeseries for regression
+    feature_values = np.array(range(len(times))).astype(float)
+    feature_ts: TimeSeries = TimeSeries.from_times_and_values(times, feature_values)
+
+    def test_autoregressive_models_runnability(self):
```

Comment:
Good idea to split into regressive and autoregressive test cases - it can be even better to do completely two different python classes - AutoregressiveModelsTestCase and RegressiveModelsTestCase